genisys/templates/preseed.cfg.jinja2

```diff
@@ -94,3 +94,10 @@ in-target chmod 644 /root/.ssh/authorized_keys; \
 in-target chmod 700 /root/.ssh/
 {% endif %}
 
+# copy firstboot.service file to client 
+d-i preseed/late_command string cp -r /ftp/first-boot/genisys-firstboot.service /etc/systemd/system/
+# enable systemd service
+d-i preseed/late_command string systemctl enable
```

Comment:
Systemctl enable needs another argument which is the service to enable. On a normal system you would:
1. Install the service file (done above)
2. Run `systemctl daemon-reload` to load the fact that there is a new service
3. Run `systemctl enable <service>` to enable it
I don't know if (2) is required here, since this is technically running during the install process, but it should be noted.